jira/src/main/kotlin/gropius/sync/jira/SyncStatus.kt

```diff
@@ -0,0 +1,58 @@
+package gropius.sync.jira
+
+import kotlinx.coroutines.reactor.awaitSingle
+import org.bson.types.ObjectId
+import org.springframework.data.annotation.Id
+import org.springframework.data.mongodb.core.index.Indexed
+import org.springframework.data.mongodb.core.mapping.Document
+import org.springframework.data.mongodb.repository.ReactiveMongoRepository
+import org.springframework.stereotype.Repository
+import org.springframework.stereotype.Service
+import org.springframework.transaction.annotation.Transactional
+import java.time.OffsetDateTime
+
+/**
+ * Base class for storing information about general sync status
+ * @param imsProject IMS project ID
+ * @param lastSuccessfulSync Timestamp of the last successful sync. nullable for future use
+ */
+@Document
+data class SyncStatus(
+    @Indexed
+    val imsProject: String,
+    @Indexed
+    var lastSuccessfulSync: OffsetDateTime?,
+) {
+    /**
+     * MongoDB ID
+     */
+    @Id
+    var id: ObjectId? = null
+}
+
+/**
+ * Repository for accessing the sync status
+ */
+@Repository
+interface SyncStatusRepository : ReactiveMongoRepository<SyncStatus, ObjectId> {
+    /**
+     * Find using the IMSProject ID
```

Comment:
missing docs for param

Reply:
2adc7f03c2a13e2368756f1bbab80527b129f499